### PR TITLE
Add Beam *spb.Entry stream reader

### PR DESCRIPTION
**Pull request by schroederc**

### CODE REVIEW

File: kythe/go/serving/pipeline/beamio/BUILD [hunk 1]
```diff
@@ -0,0 +1,29 @@
+load("//tools:build_rules/shims.bzl", "go_library", "go_test")
+
+package(default_visibility = ["//kythe:default_visibility"])
+
+go_library(
+    name = "beamio",
+    srcs = glob(
```
Comment (creachadair) [inline kythe/go/serving/pipeline/beamio/BUILD]: There's only one! Please let's just list it.

File: kythe/go/serving/pipeline/beamio/entries.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ */
+
+package beamio
+
+import (
+	"context"
+
+	"kythe.io/kythe/go/storage/stream"
+
+	spb "kythe.io/kythe/proto/storage_go_proto"
```
Comment (creachadair) [inline kythe/go/serving/pipeline/beamio/entries.go]: I think we wound up putting proto imports at the end?

Comment (schroederc) [reply kythe/go/serving/pipeline/beamio/entries.go]: ✓

File: kythe/go/serving/pipeline/beamio/BUILD (hunk 1 above)
Comment (schroederc) [reply kythe/go/serving/pipeline/beamio/BUILD]: ✓

**Pull request merged**